components/seo/seo.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Meta } from '../meta/meta-service';
+
+export type PayloadDirectives = {
+	deindex?: boolean;
+};
+
+export class Seo {
+	static processPayloadDirectives(data: PayloadDirectives) {
+		if (data.deindex) {
+			console.log('Deindexing page');
```

Comment:
Keep this console log?

Reply:
Yeah I wanted it around for a bit. i'll add a todo to remove it

Reply:
actually, could just do it locally when i test i guess.